synapse/rest/media/v1/upload_resource.py

```diff
@@ -88,8 +90,9 @@ async def _async_render_POST(self, request: Request) -> None:
         # TODO(markjh): parse content-dispostion
 
         try:
+            content = request.content  # type: IO  # type: ignore
```

Comment:
The type of this is undefined right now in Twisted (I think)?

Reply:
Why the `type: ignore` after the `IO` type definition?

Reply:
It overrides the definition previously calculated.